src/dubinsmaneuver3d.jl

```diff
@@ -99,18 +99,59 @@ function DubinsManeuver3D(qi::Vector{Float64}, qf::Vector{Float64},
     return maneuver
 end
 
-function compute_sampling(self::DubinsManeuver3D; numberOfSamples::Integer = 1000)
+"""
+    get_configuration(self::DubinsManeuver3D, distance::Number)
+
+Finds vehicle configuration at the given distance from start of maneuver
+    * self - maneuver on which a configuration is queried
+    * distance - distance from the start of maneuver where vehicle configuration is queried
+"""
+function get_configuration(self::DubinsManeuver3D, distance::Number)
+    Dlat, Dlon = self.path
+    lena = Dlon.maneuver.length
+    qSZ = getCoordinatesAt(Dlon, distance)
+    qXY = getCoordinatesAt(Dlat, qSZ[1])
+    return [qXY[1], qXY[2], qSZ[2], qXY[3], qSZ[3]]
+end
+
+"""
+    compute_sampling(self::DubinsManeuver3D; numberOfSamples::Number = 1000)
+
+Sample the maneuver with a given number of samples
+    * self - maneuver to be sampled
+    * numberOfSamples - number of samples taken along the maneuver
+"""
+function compute_sampling(self::DubinsManeuver3D; numberOfSamples::Number = 1000)
+    Dlat, Dlon = self.path
+    lena = Dlon.maneuver.length
+    samplingStep = lena/(numberOfSamples-1)
```

Comment:
This is not numerically stable. The sampling has been fixed in [this commit](https://github.com/comrob/Dubins3D.jl/commit/60e41678816337e8feef930f89dee2327636c233).